<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pixelSelection</class>
 <widget class="QWidget" name="pixelSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="closeButtonPB">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>700</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>681</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="UpLeftTab">
    <attribute name="title">
     <string>Upper Left</string>
    </attribute>
    <widget class="QWidget" name="UpperLeftW" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>421</width>
       <height>641</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="UpRightTab">
    <attribute name="title">
     <string>Upper Right</string>
    </attribute>
    <widget class="QWidget" name="UpperRightW" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>421</width>
       <height>641</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="DownLeftTab">
    <attribute name="title">
     <string>Lower Left</string>
    </attribute>
    <widget class="QWidget" name="LowerLeftW" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>421</width>
       <height>641</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="DownRightTab">
    <attribute name="title">
     <string>Lower Right</string>
    </attribute>
    <widget class="QWidget" name="LowerRightW" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>421</width>
       <height>641</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="drawPB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>700</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
